src/pages/TodoApp/TodoApp.jsx

```diff
@@ -22,12 +23,18 @@ const validate = values => {
     return error;
 }
 
+const FORM_NAME = 'newTodoForm';
+
 const ToDoApp = ({ handleSubmit }) => {
     const dispatch = useDispatch();
+    const disabled = useSelector(state => state.todosReducer.disabled);
```

Comment:
const selector = state => state.todosReducer;
const { disabled } = useSelector(selector);

Reply:
in reducer